Source/Core/Src/UnPlat.cpp

```diff
@@ -487,6 +487,7 @@ void appInit()
 	GProcessorCount = SDL_GetCPUCount();
 #endif // PLATFORM_
 
+#if __INTEL__
```

Comment:
This is not particularly necessary, as `FGlobalPlatform_CPUID` already does nothing on non-x86 platforms.